bindings/wp8/MegaSDK.cpp

```diff
@@ -3439,3 +3470,24 @@ void MegaSDK::freeTransferListener(DelegateMTransferListener *listener)
 	LeaveCriticalSection(&listenerMutex);
 	delete listener;
 }
+
+MegaLogger *MegaSDK::createDelegateMLogger(MLoggerInterface^ logger)
+{
+    if (logger == nullptr) return NULL;
+
+    DelegateMLogger *delegateLogger = new DelegateMLogger(logger);
+    EnterCriticalSection(&loggerMutex);
+    activeLoggers.insert(delegateLogger);
+    LeaveCriticalSection(&loggerMutex);
+    return delegateLogger;
+}
+
+void MegaSDK::freeLogger(DelegateMLogger *logger)
+{
+    if (logger == nullptr) return;
+
+    EnterCriticalSection(&loggerMutex);
+    activeLoggers.erase(logger);
+    LeaveCriticalSection(&loggerMutex);
```

Comment:
Maybe it's not necessary to erase the item from the vector, since it's already erased in `MegaSDK::removeLoggerObject()`